src/chart/ChartUtils.ts

```diff
@@ -164,10 +164,14 @@ export const downloadCSV = (rows) => {
     headers.forEach((header) => {
       // Parse value
       let value = row[header];
-      if (value && 'low' in value) {
+      if (value?.low !== undefined) {
         value = value.low;
       }
-      csv += `${JSON.stringify(value)}`;
+      if(typeof value === 'string'){
+        csv += value;
+      }else{
+        csv += `${JSON.stringify(value)}`;
+      }
```

Comment:
Hi @JonanOribe, I think we should keep "stringified strings", in order to avoid this phenomena when string value contains delimiter characters

Non stringified version:
![Capture d’écran 2025-01-28 à 15 39 09](https://github.com/user-attachments/assets/fc9e131c-33f3-408a-8fb7-ff940b50c353)

Stringified version:
![Capture d’écran 2025-01-28 à 15 38 43](https://github.com/user-attachments/assets/7a2a3e56-9db9-4d1c-a647-22e38fe0c7bc)